core/server/api/v2/slack.js

```diff
@@ -0,0 +1,12 @@
+const common = require('../../lib/common');
+
+module.exports = {
+    docName: 'slack',
+    sendTest: {
+        permissions: false,
+        query() {
+            common.events.emit('slack.test');
+            return Promise.resolve();
```

Comment:
```suggestion
```

Had a second thought on this part. We don't have to return anything from the query function in this situation, would remove unneeded `return Promise.resolve();`